CONTRIBUTING.md

```diff
@@ -53,6 +53,37 @@ bin/cuke [<folder>...]
 ```
 
 
+## Pull Requests
+
+Each pull request (PR) should mention the [issue](https://github.com/Originate/git-town/issues)
+it implements in the PR description:
```

Comment:
1. "resolves" sounds better than "implements" here
2. ... in its description (less wordy)